### PR TITLE
docs(nxdev): customize webpack guide

### PR DESCRIPTION
**Pull request by isaacplmann**
Customize webpack guide

### CODE REVIEW

File: docs/shared/guides/customize-webpack.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Note: For Angular developers, use an executor like [`ngx-build-plus`](https://github.com/manfredsteyer/ngx-build-plus).
+
+In your `project.json` configuration for the `@nrwl/web:build` or `@nrwl/node:build` executor, set the [`webpackConfig`](/web/build#webpackconfig) option to point to your custom webpack config file. i.e. `apps/my-app/custom-webpack.config.js`
+
+The custom webpack file contains a function that takes as input the existing webpack config and then returns a modified config object. `context` includes all the options specified for the executor.
+
+`apps/my-app/custom-webpack.config.js`:
+
+```typescript
+module.exports = (config, context) => {
```
Comment (jaysoo) [inline docs/shared/guides/customize-webpack.md:14]: Should  we also mention using `webpack-merge` to merge configs? And how to look up specific loaders.

Might also want to pass more things on the context so users don't need to find things like `babel-loader` themselves (but this requires code changes).

Comment (jaysoo) [reply docs/shared/guides/customize-webpack.md:14]: And maybe provide some basic examples of adding a loader, or changing some base webpack config.

Comment (isaacplmann) [reply docs/shared/guides/customize-webpack.md:17]: what kinds of things do people try to do?  I personally avoid webpack configs as much as possible

Comment (EfstathiadisD) [reply docs/shared/guides/customize-webpack.md:17]: Webpack can be used for many things. We do loads of stuff with it such as copying files for production builds passed from CI, performance optimization and more. A couple examples would be appreciated.

**Pull request merged**